src/Features/Core/Portable/EncapsulateField/AbstractEncapsulateFieldService.cs

```diff
@@ -265,26 +266,26 @@ private async Task<Solution> EncapsulateFieldsInCurrentProcessAsync(Document doc
             if (field.IsReadOnly)
             {
                 // Inside the constructor we want to rename references the field to the final field name.
-                var constructorLocations = GetConstructorLocations(field.ContainingType);
+                var constructorLocations = GetConstructorLocations(solution, field.ContainingType);
```

Comment:
previously, we were operating on data that used the Location type.  However, this type references a SyntaxTree, which is not something we marshall to/from oop.  Now it uses DocumentId/TextSpan to match the data we have in this lightweight type.